releases/cosmwasm/2025-02-xrplevm-GMP-v6.4.0.md

```diff
@@ -0,0 +1,351 @@
+# Ripple EVM Sidechain GMP v6.4.0
+
+|                | **Owner**                                                                   |
+| -------------- | --------------------------------------------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io>                                      |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io>, @talalashraf <talal@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
```

Comment:
This was deployed (by Peersyst) on February 5

Reply:
that's only a test connection. We'll be connecting the official testnet to our devnet-amplifier/stagenet as well, similar to how Flow is done